Description: Enable automake maintainer mode
 Enable AM_MAINTAINER_MODE to stop make from rebuilding configure in
 case of timestamp skew (as created by dpkg-source)
Forwarded: no
Author: Floris Bruynooghe <flub@devork.be>

--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,8 @@
 AM_INIT_AUTOMAKE([plotutils],[2.6])
 AM_CONFIG_HEADER(config.h:config.hin)
 
+AM_MAINTAINER_MODE
+
 # The following will be defined in config.h, if appropriate.
 
 # Miscellaneous.
@@ -64,6 +66,7 @@
 # Note: this can apparently alter CFLAGS, on a few platforms, e.g., on
 # SCO OpenServer 5 (i.e. *-*-sco3.2v5*), "-belf" is added.
 AC_PROG_LIBTOOL
+AC_PROG_CXX
 
 # Determine extension (e.g. ".exe") on executables, if any.
 AC_EXEEXT
